style.css

```diff
@@ -0,0 +1,268 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+body {
+    margin: 0 auto;
+    height:100%;
+}
+nav {
+    margin: 0;
+    padding: 0;
+    background-color: #111111;
+    width: 100%;
+    position: sticky;
```

Comment:
this has very bad support:
https://caniuse.com/#feat=css-sticky
use position fixed, you dont need sticky for this as its always at the top

Reply:
changed in 6ac4215